examples/msbr/msbr.serpent

```diff
@@ -51,9 +51,9 @@ therm gr 900 gre7.18t gre7.20t  % 900K C-nat
 set title "MSBR Saltproc long-term, 91% removal, BOL, fresh fuel, ENDF"
 
 
-set acelib "/home/andrei2/serpent/xsdata/endfb7/sss_endfb7.xsdata"
-set declib "/home/andrei2/serpent/xsdata/endfb7/sss_endfb7.dec"
-set nfylib "/home/andrei2/serpent/xsdata/endfb7/sss_endfb7.nfy"
+set acelib "sss_endfb7.xsdata"
+set declib "sss_endfb7.dec"
+set nfylib "sss_endfb7.nfy"
```

Comment:
You don't have any issues with finding these files? Would this need to be changed if it were run on a different machine?

Reply:
These files are discoverable if the user sets their path variables correctly, which they'll need to do if they want serpent to run properly. I think I added a section in the docs pointing to the serpent wiki, but it may be worth repeating on our own docs.

Reply:
I've added some details about this in issue #109